From b387ea83b01e6a5082eed9a3e1c688e2841aa719 Mon Sep 17 00:00:00 2001
From: roqqit <roqqit@proton.me>
Date: Tue, 26 Nov 2024 11:23:44 -0800
Subject: [PATCH] [doged] Fix wallet test fee (#90)

maxfee exceeds the coinbase value

ninja check is now green

diff --git a/src/wallet/test/wallet_tests.cpp b/src/wallet/test/wallet_tests.cpp
index 9cb4f303bf53d30f776bd2c05e65092efda8b29e..5d763ec9b92c7b89f36af2cf96d8ac82879fe9f9 100644
--- a/src/wallet/test/wallet_tests.cpp
+++ b/src/wallet/test/wallet_tests.cpp
@@ -64,8 +64,7 @@ static CMutableTransaction TestSimpleSpend(const CTransaction &from,
                                            uint32_t index, const CKey &key,
                                            const CScript &pubkey) {
     CMutableTransaction mtx;
-    mtx.vout.push_back(
-        {from.vout[index].nValue - DEFAULT_TRANSACTION_MAXFEE, pubkey});
+    mtx.vout.push_back({from.vout[index].nValue - DEFAULT_PAY_TX_FEE, pubkey});
     mtx.vin.push_back({CTxIn{from.GetId(), index}});
     FillableSigningProvider keystore;
     keystore.AddKey(key);
